### DISCUSSION TITLE
Keepinv/keep inventory

### DISCUSSION THREAD

**Discussion by Arkkanus** [Zrips/CMI#2296] [Q&A]
**Question: **
I recently purchased CMI and have most everything set up the way i want. But am having an issue where i would like top use the cmi.keepinventory permission as a toggle for players so they can choose weather they want to play with it enabled or not, similar to /fly. 
 
 I've seen that you can use a sign or item to execute commands but not seeing a way to enable this as a feature to use with alias'. I am using luckyperms. if this helps in any way.

**Comment by mercurialmusic** [comment]
You could do this with kits, interactive commands, or holograms, with two separate blocks/holograms with commands like so:

The first to give them the perm:
asConsole! lp user [playerName] permission set cmi.keepinventory true server=global world=global

The second to remove the perm:
asConsole! lp user [playerName] permission unset cmi.keepinventory server=global world=global

Note that these by themselves would not give a feedback message. To do that you could set up kits containing those commands along with secondary commands to send the feedback message, then point the IC commands to run the kit asConsole! (if you don't want to give them permission for the kit).

Since we don't currently have if-then-else functionality (which would be lovely) I'm not sure that you could set this up as a true toggle with just one block.

**Comment by sandwally** [comment]
You can also do a custom alias.

  keepinv on:
    Cmds:
    - fromConsole! lp user [playerName] permission set cmi.keepinventory true server=global world=global
    - fromConsole! tellraw {player} {"text":"&fYou will keep your inventory on death"}

  keepinv off:
    Cmds:
    - fromConsole! lp user [playerName] permission unset cmi.keepinventory server=global world=global
    - fromConsole! tellraw {player} {"text":"&fYour items will drop on the ground on death"}

Just remember the spacings, github didn't want to include it.

**Comment by CloudeLecaw** [comment]
would be nice if you close the ticket ;)
